### DISCUSSION TITLE
Fit any kind of atomic tensor with deepMD

### DISCUSSION THREAD

**Discussion by Fr2eZ** [deepmodeling/deepmd-kit#2323] [Q&A] [answered]
Dear all, 

I would like to use DeepMD to fit shielding and efg tensors to predict NMR and NQR parameters. 
Is it possible to fit them using the same framework as for polarizability ?

**Comment by wanghan-iapcm** [comment, marked as answer]
Yes please refer to the work https://doi.org/10.1039/D0CP01893G and the doc https://docs.deepmodeling.com/projects/deepmd/en/master/model/train-fitting-tensor.html#fit-tensor-like-dipole-and-polarizability
